run_benchmark_slurm.sh

```diff
@@ -20,6 +20,15 @@ else
       export DOMAINLAB_CUDA_START_SEED=$2
 fi
 
+# Extract output_dir from the YAML configuration file
```

Comment:
could you put the common bash operations into a function in a separate script in scripts folder?